src/main/java/core/basesyntax/service/impl/BalanceAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class BalanceAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line at the start of the `apply` method.